### PR TITLE
Add: set day time and change weather

### PR DESCRIPTION
**Pull request by SOF-Ice**
- Add: Possibility to change Time and Weather for officer only (@TomDraal, @SOF-Ice)

**When merged this pull request will:**
- Add addaction for officer **_only_** on the btc_gear_object (arsenal)
- Set day or night without "passing time" with Zeus module
- Change Weather
- Add new container in stringtable.xml with french traduction

**Final test:**
- [x] local
- [ ] server

### CODE REVIEW

**Comment by SOF-Ice** [comment]
Sorry i made a mistake by including two of yours commits...
Don't know how to change this ...

**Review by Vdauphin — CHANGES_REQUESTED** [review]
Hello,
Please add a setting for each feature
Some admin will not want this
Cheers

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf [hunk 1]
```diff
@@ -228,3 +228,12 @@ if (btc_p_flag > 1) then {
     }, {getForcedFlagTexture _target isNotEqualTo ""}] call ace_interact_menu_fnc_createAction;
     [player, 1, ["ACE_SelfActions", "ACE_Equipment"], _action] call ace_interact_menu_fnc_addActionToObject;
 };
+
+//Change day time and weather
+
+_action = ["set_day", localize "STR_BTC_HAM_ACTION_SET_DAY", "\A3\ui_f\data\igui\cfg\simpleTasks\types\default_ca.paa", {((6 - daytime + 24) % 24) remoteExec ["skipTime", 0]}, {player getVariable ["side_mission", false]}] call ace_interact_menu_fnc_createAction;
```
[the remark this answers is not in the document]
Comment (Vdauphin) [reply =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf:234]: Target the server. Not cclient as `skipTime` is server exec

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf [hunk 2]
```diff
@@ -228,3 +228,12 @@ if (btc_p_flag > 1) then {
...
+[btc_gear_object, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
+_action = ["set_night", localize "STR_BTC_HAM_ACTION_SET_NIGHT", "\A3\ui_f\data\igui\cfg\simpleTasks\types\default_ca.paa", {((22 - daytime + 24) % 24) remoteExec ["skipTime"], 0}, {player getVariable ["side_mission", false]}] call ace_interact_menu_fnc_createAction;
+[btc_gear_object, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
+_action = ["set_weather", localize "STR_BTC_HAM_ACTION_CHANGE_WEATHER", "\A3\ui_f\data\igui\cfg\simpleTasks\types\default_ca.paa", { 5 setrain 0; 5 setfog 0; 5 setOvercast 0; 10 setRainbow 1; forceWeatherChange;}, {player getVariable ["side_mission", false]}] call ace_interact_menu_fnc_createAction;
```
[the remark this answers is not in the document]
Comment (Vdauphin) [reply =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf:238]: Also please break the code section of the action in multiple line for readability.

**Pull request closed without merging**